packages/s/soloud/xmake.lua

```diff
@@ -0,0 +1,62 @@
+-- Examples:
+-- https://github.com/xmake-io/xmake-repo/tree/master/packages/n/newtondynamics
+-- https://github.com/xmake-io/xmake-repo/blob/dev/packages/z/zlib/xmake.lua
+-- https://github.com/tboox/benchbox
+
+package("soloud")
+    set_description("SoLoud is an easy to use, free, portable c/c++ audio engine for games.")
+    
+    add_urls("https://sol.gfxile.net/soloud/soloud_20200207_lite.zip")
```

Comment:
Please put urls and versions after homepage and license

Reply:
Any reason for not using the official repo?

Reply:
I'm not sure where the `sol.gfxile.net` URL came from. Instead of changing the link to `https://solhsa.com/soloud/soloud_20200207_lite.zip`, I went ahead and changed it to the GitHub release zip.

Reply:
is this https://github.com/jarikomppa/soloud?

why not use github url and releases?